jpsonic-main/src/main/java/com/tesshu/jpsonic/controller/StatusChartController.java

```diff
@@ -184,8 +191,12 @@ public ModelAndView handleRequest(HttpServletRequest request, HttpServletRespons
         rangeAxis.setTickMarkPaint(fgColor);
         rangeAxis.setAxisLinePaint(fgColor);
 
-        synchronized (LOCK) {
+        try {
```

Comment:
JFreeChart's drawing itself is unthread safe, but since the dataset is local, I don't think there will be any problems 🙄 The past use of synchronized seems to be a misunderstanding of what causes IO errors. It's not a super important feature, so it will probably be rewritten and monitored.